aspnetcore/signalr/hubs.md

```diff
@@ -23,6 +23,9 @@ To register the services required by SignalR hubs, call <xref:Microsoft.Extensio
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/signalr/hubs/samples/6.x/SignalRHubsSample/Program.cs" id="snippet_AddSignalR" highlight="4":::
 
+> [!NOTE]
```

Comment:
Why is this only in .NET 6? Why is .NET 7 at the bottom, it needs to be on top.

Reply:
**"Why is this only in .NET 6? Why is .NET 7 at the bottom, .NET 7 needs to be on top."**
* Looking at the history, Brennan added 7.0 at the bottom in April.  
* Later you left .NET 7 in the bottom in [your update in the .NET 7 version in July] (https://github.com/dotnet/AspNetCore.Docs/commit/cc7a860efe9d3b866871ce83c5e533e67b36feec)

In my draft, I unfortunately I didn't notice the 7.0 at the bottom and thought 6.0 at the top was the latest and edited that.  I should have looked around further, however, I know better to watch out for them.  I have seen several topics where the latest was added to the bottom.  When I run into those, I would move them to the top.  As I will this one now.